pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -81,8 +81,8 @@ class MindtPySolver(object):
 
     This solver implementation has been developed by David Bernal <https://github.com/bernalde>
     and Zedong Peng <https://github.com/ZedongPeng> as part of research efforts at the Grossmann
-    Research Group (http://egon.cheme.cmu.edu/) at the Department of Chemical Engineering at
-    Carnegie Mellon University.
+    Research Group (http://egon.cheme.cmu.edu/) and the Benal Research Group (https://bernalde.github.io/)
```

Comment:
I think you meant the "Bernal" Research Group in which case this typo appears in multiple places.